pyfr/backends/openmp/kernels/errest.mako

```diff
@@ -3,37 +3,58 @@
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 
 void
-errest(int nrow, int nblocks, fpdtype_t *__restrict__ error,
+errest(int nrow, int nblocks, fpdtype_t *__restrict__ reduced,
+% if method == 'errest':
        fpdtype_t *__restrict__ x, fpdtype_t *__restrict__ y,
        fpdtype_t *__restrict__ z, fpdtype_t atol, fpdtype_t rtol)
+% else:
+       fpdtype_t *__restrict__ rcurr, fpdtype_t *__restrict__ rold,
+% if dtau == 'scalar':
+       fpdtype_t dtau)
+% else:
+       fpdtype_t *__restrict__ dtau)
```

Comment:
dt is probably a better name (if you consider this to be a generic residual kernel).